sn_node_manager/src/bin/daemon/main.rs

```diff
@@ -72,9 +66,42 @@ impl SafeNodeManager for SafeNodeManagerDaemon {
 
         Ok(Response::new(NodeServiceRestartResponse {}))
     }
+
+    async fn nodes_info(
+        &self,
+        request: Request<NodesInfoRequest>,
+    ) -> Result<Response<NodesInfoResponse>, Status> {
+        println!("RPC request received {:?}", request.get_ref());
+        info!("RPC request received {:?}", request.get_ref());
+        let node_registry = Self::load_node_registry().map_err(|err| {
+            Status::new(
+                Code::Internal,
+                format!("Failed to load node registry: {err}"),
+            )
+        })?;
+
+        let nodes_info = node_registry
+            .nodes
+            .iter()
+            .map(|node| Node {
+                peer_id: node.peer_id.map(|id| id.to_bytes()),
+                status: node.status.clone() as i32,
+                number: node.number as u32,
+            })
+            .collect::<Vec<_>>();
+        Ok(Response::new(NodesInfoResponse { nodes: nodes_info }))
```

Comment:
I think it would be better to call this command `Status`, corresponding with the command on the node manager.

Reply:
Updated! Thank you!